_includes/releases/v21.2/v21.2.15.md

```diff
@@ -0,0 +1,49 @@
+## v21.2.15
+
+Release Date: August 29, 2022
+
+{% include releases/release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v21-2-15-sql-language-changes">SQL language changes</h3>
+
+- Removed the `DatabaseID` field from the sampled query telemetry log due to its potential to cause indefinite blocking in the case of a lease acquisition failure. [#85022][#85022]
+
+<h3 id="v21-2-15-operational-changes">Operational changes</h3>
+
+- Telemetry logs now display more finely redacted error messages from SQL execution. Previously, the entire error string was redacted. [#85463][#85463]
+
+<h3 id="v21-2-15-command-line-changes">Command-line changes</h3>
+
+- The CLI now contains a flag (`--log-config-vars`) that allows for environment variables to be specified for expansion within the logging configuration file. This allows a single logging configuration file to service an array of sinks without further manipulation of the configuration file. [#85172][#85172]
+- [`cockroach debug zip`](../v21.2/cockroach-debug-zip.html) now includes all system tables by default, except for a few (a deny list). [#86142][#86142]
```

Comment:
Betting I'll need to track down related changes to https://www.cockroachlabs.com/docs/v21.2/cockroach-debug-zip.html#files then. I'll handle it.